ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -96,35 +93,35 @@ public extension Colors {
     }
 
     // Objective-C support
-    @objc static var tableBackground: UIColor { return Table.background }
-    @objc static var tableBackgroundGrouped: UIColor { return Table.backgroundGrouped }
-    @objc static var tableCellBackground: UIColor { return Table.Cell.background }
-    @objc static var tableCellBackgroundGrouped: UIColor { return Table.Cell.backgroundGrouped }
-    @objc static var tableCellBackgroundSelected: UIColor { return Table.Cell.backgroundSelected }
-    @objc static var tableCellImage: UIColor { return Table.Cell.image }
+    @objc static var tableBackground: UIColor { return surfacePrimary }
```

Comment:
With tokens in place, these properties should probably be deprecated.
@mischreiber going forward what's the best practice for the client code that wants to retrieve these token values? Instantiate a default TableViewCellTokens and query the desired token property?

Reply:
Makes sense. @rdeassis going forward I think you're probably correct, though that's a pretty significant change (especially as it pertains to all the stuff in `Colors.swift`).